### PR TITLE
🚀 Release v1.8.3

### PR DESCRIPTION
**Pull request by chandankumar4**
**What this PR does / why we need it**:
- Release note for v1.8.3

/area release

### CODE REVIEW

File: CHANGELOG/v1.8.3.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+- Management Cluster: v1.27.x -> v1.31.x
+- Workload Cluster: v1.25.x -> v1.31.x
+
+[More information about version support can be found here](https://cluster-api.sigs.k8s.io/reference/versions.html)
+
+## Changes since v1.8.2
+## :chart_with_upwards_trend: Overview
+- 1 new commit merged
+
+## :seedling: Others
```
Comment (sbueringer) [inline CHANGELOG/v1.8.3.md:12]: Let's add: 
* https://github.com/kubernetes-sigs/cluster-api/pull/11154 (merged today)
* https://github.com/kubernetes-sigs/cluster-api/pull/11157 (merged today)
* https://github.com/kubernetes-sigs/cluster-api/pull/11164 (will merge today)

Comment (chandankumar4) [reply CHANGELOG/v1.8.3.md:16]: Added

**Pull request merged**